.github/workflows/ci-linux_mac.yml

```diff
@@ -56,6 +73,9 @@ jobs:
         os:
           - ${{ inputs.matrix_image }}
     runs-on: ${{matrix.os}}
+    #value: quay.io/pypa/manylinux2014_x86_64:2022-11-06-7be974c
+    # modified from above tag to add linux-headers
+    container: ${{inputs.manylinux && 'ghcr.io/ihnorton/tiledb-manylinux2014_x86_64:2023-04-02' || ''}}
```

Comment:
This image is (should be..) temporary. IIRC the reason for this customized image was to install some vcpkg dependency, but we are now doing that below anyway: https://github.com/TileDB-Inc/TileDB/pull/4491/files#diff-214883620bda15d306d05ffe97a2adca8579ee180118160c9295514c3d933cc6R112

We should use a more-recent upstream if at all possible, or if still necessary, put the customized image in a package registry owned by TileDB-Inc.

Reply:
Is the Dockerfile for your image available somewhere?